### PR TITLE
Docs: Update fares examples

### LINKED ISSUE #52: Explain what the difference between Fares v1 and Fares v2 is under the GTFS Schedule section
## **Content request**

**Problem**
Now that Fares v1 and Fares v2 can both be used in the spec, and because Fares v2 is not based on top of Fares v1, it's confusing for someone that is just starting with fares information to understand
- What is the difference between the two?  
- Which one should I use?
- Is Fares v2 completely adopted, meaning that what is displayed in the Fares v2 example is everything this extension can do?

**Proposed solution**
The first page of the [Google Doc](https://docs.google.com/document/d/19j-f-wZ5C_kYXmkLBye1g42U-kvfSVgYLkkG5oyBauY/edit#heading=h.mhjuwqk2h5r3) contains two paragraphs that explain:
- why did we create Fares v2
- what is the difference between the two

I suggest we append the examples existing on gtfs.org ([here](https://gtfs.org/schedule/examples/fares-v1/) and [here](https://gtfs.org/schedule/examples/fares-v2/)) with these pieces of information, as well as mention that the example shows for Fares v2 is part of a bigger extension project. 

**Alternative solutions**
Rely on the Google Doc to answer these questions. 
This Google Doc can be found on the [Extensions](https://gtfs.org/extensions/) page, which is not the first place people go if what they are trying to do is implement fares data based on the official spec.

### PR DESCRIPTION
**Pull request by isabelle-dr**
Closes #52.
This is what I wish I had when I started working on this 🙈.
I took this content from the [official proposal](https://docs.google.com/document/d/19j-f-wZ5C_kYXmkLBye1g42U-kvfSVgYLkkG5oyBauY/edit#heading=h.m7bmyhov41du) so that people that are just starting with fares can quickly understand:
- that fares v2 is _not_ built on top of fares v1
- that the example here is just what has been adopted and that fares v2 adoption is not finished.

### CODE REVIEW

**Comment by isabelle-dr** [comment]
@bdferris-v2 FYI

File: docs/schedule/examples/fares-v1.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 <hr>
 
+Composed of [fare_attributes.txt](../../reference/#fare_attributestxt) and [fare_rules.txt](../../reference/#fare_rulestxt), Fares v1 is the current official method for describing fare information in GTFS. However the two files are limited in the breadth of factors they can efficiently describe, and are ambiguous to implement. 
+[Fares v2](../../examples/fares-v2/) is an extension project under active development, and it aims to address the limitations of Fares v1.
```
Comment (emmambd) [inline docs/schedule/examples/fares-v1.md:10]: ``Fares v1 is the current official method for describing fare information in GTFS`` - is this accurate given the base implementation of Fares v2 being adopted? All other explanation is clear, but this confuses me. @isabelle-dr

**Review by emmambd — COMMENTED** [review]
@isabelle-dr This is definitely the clearest description of fares v1 vs. fares v2 that I have seen! I've added one comment about how we describe the stage fares v2 is in vs. fares v1.

File: docs/schedule/examples/fares-v1.md (hunk 1 above)
Comment (isabelle-dr) [reply docs/schedule/examples/fares-v1.md:10]: Good point, since we have Fares v2 now the two options are available. I'll modify accordingly

Comment (isabelle-dr) [reply docs/schedule/examples/fares-v1.md:10]: Attempt to solve this in [this commit](https://github.com/MobilityData/gtfs.org/pull/58/commits/d4559ebaea43812c1b06ab976544f96fb30ee41a)

**Comment by isabelle-dr** [comment]
I gave a shot at solving the point you raised, please take another look 😊

File: docs/schedule/examples/fares-v1.md (hunk 1 above)
Comment (emmambd) [reply docs/schedule/examples/fares-v1.md:10]: Much clearer, thank you!

**Review by emmambd — APPROVED** [review]

**Pull request merged**